### DISCUSSION TITLE
How to register manual sulcal labelling to MNI152 Space?

### DISCUSSION THREAD

**Discussion by jim81098** [brainvisa/brainvisa.github.io#183] [Q&A]
Dear all,
I am new to using BrainVISA and need some guidance on a project. We aim to visualise a single sulcus's variability within a few hundred subjects, by computing some kind of probability map in the MNI152 space, using manual labelling. However, we're unsure how to register either the sulcal graph or a nifti volume resulting from the graph into the MNI152 space.

Can we compute the non-linear registration to the MNI space using SPM (maybe SPM Normalization Pipeline under Morphologist > Registration) using the original T1 and then apply the resulting transformation to a simplified nifti in the same space as the original T1?

Or is there a more appropriate method within BrainVISA to achieve this?

Thank you in advance for any advice or suggestions!

**Comment by denisri** [comment]
Hi,
What you want to do looks like what the process "Volumic sulcus SPAM" does (in Morphologist/morphometry/Sulci/SPAM), except 2 points:
- in our process the images are not registered in MNI space but in our "Talairach" one. There is a single simple affine matrix between them. We can re-write the result in MNI space if needed (at the cost of an additional resampling), or preferably we should add a parameter to the process to specify the output space.
- we only use an affine registration to the common space (it's a SPM normalization to MNI actually, combined with the affine matrix I mention just above). We do not use any non-linear deformation. We fear non-linear warping would erase the inter-individual variability that we want to characterize. If you need non-linear registration, then you will need to register non-linearly the T1 MRI (using SPM, or the SPM toolbox in brainvisa), use the process individually for each subject, then apply the warping to it, and afterwards add all subjects.
Denis

**Reply by jim81098** [reply]
Thank you very much for your reply.  We have opted to proceed with the first option and have successfully registered it into Talairach space with the Volumic sulcus SPAM. Could you kindly demonstrate how to rewrite the result in MNI space as you suggested?

**Reply by denisri** [reply]
Hi,
To convert the sulcus SPAM image to MNI space, you may use this transformation matrix: save it in a text file with the `.trm` extension, for instance `icbm_TO_icbm_template.trm`
```
96 96 114
-1 -0 -0
-0 -1 -0
-0 -0 -1
```
(this transformation just adjusts for the ICBM2009c template field of view, and we will add it in brainvisa)
```
AimsApplyTransform -i <sulci_spam>.nii.gz -o <sulci_spam_mni>.nii.gz -d /casa/install/share/brainvisa-share-5.1/transformation/talairach_TO_spm_template_novoxels.trm -d ~/data/icbm_to_icbm_template.trm --input-coords acpc
```
The next release of Brainvisa will allow to avoid the needs of this matrix, but with the current release, you will need it.

**Reply by jim81098** [reply]
Hi, 
Thank you so much for the prompt reply. I assume this template is the ICBM 2009c Nonlinear Asymmetric template? What if we want to also use the ICBM 2009c Nonlinear Symmetric template, how should we change the transformation file? Thank you in advance!

**Reply by denisri** [reply]
I have not looked at it closely but I assume the differences between the symmetric and non-symmetric ICBM templates are only local non-linear deformations, and that the global affine is the same, but I'm not totally sure of that. However if I'm right, using the same affine transformation we use here will not reduce the accuracy.
Denis
